management/baremetal.md

```diff
@@ -1,12 +1,13 @@
-##Preparing baremetal or personal VM environments for Cluster Management
+##Baremetal Server or Virtual Machine Installation
 
-This document goes through the steps to prepare a baremetal or personal VM setup for cluster management.
+This document goes through the steps to install Contiv Cluster to a baremetal
+or virtual machince.
 
 **Note:**
-- Unless explicitly mentioned all the steps below are done by logging into the same host. It is referred as *control host* below.
-- Right now we test cluster manager on Centos7.2. More OS variations shall be added in future.
+- Unless explicitly mentioned all the steps below are done by logging into the same host.
```

Comment:
I think we shall keep the line `It is referred as *control host* below.`, this term is used in step 0 below